app/src/main/java/com/iitp/njack/iitp_connect/core/home/HomeActivity.java

```diff
@@ -52,6 +64,24 @@ protected void onCreate(Bundle savedInstanceState) {
         binding.navView.setNavigationItemSelectedListener(this);
 
         drawerNavigator = new DrawerNavigator(this, authViewModel);
+
+
```

Comment:
too many blank lines.

Reply:
'java util.collections' is never used,can be removed.